### DISCUSSION TITLE
Improvement: GitHub Wiki - Getting Started

### DISCUSSION THREAD

**Discussion by Dionmok** [electricitymaps/electricitymaps-contrib#5356] [Ideas] [answered]
I believe that this section will help direct new contributors to resources to download and learn Git and GitHub. These resources were nice to know, as a novice at Git and GitHub.  Here is a link and capture of ideas for this addition to the documentation : [My Revisions](https://github.com/electricitymaps/electricitymaps-contrib/wiki/Getting-started/38348d7e54c5576e550017f803c924fbf762dfcb) 

<img width="615" alt="image" src="https://user-images.githubusercontent.com/91397067/236010351-cdb07f39-d422-40d5-8e79-3e4d9e6d3a29.png">

Thanks Again!

**Comment by VIKTORVAV99** [comment, marked as answer]
I appreciate you reverting the changes so they can be discussed first, so thanks for that!

I am not sure how much detail we want to go into here about setting up git but I suppose a link to the GitHub documentation about the subject wouldn't hurt.

I'd just want to point out at few things though, there is no GitHub to download, GitHub Desktop is completly optional and not needed to contribute to GitHub so I'd leave that one out.

I'd also use the https://docs.github.com/en/get-started/quickstart for the GitHub Docs link as well, that should cover all the GitHub basics. 

I'd also skip the git download and git documentation pages, at least the git download page. The GitHub Docs should cover all that with the above change.

**Reply by Dionmok** [reply]
Great, thank you for taking time out of your day to give me input on this!
